src/icp_canister_backend/tests/test.rs

```diff
@@ -101,7 +101,7 @@ fn test_deposit_flow() {
     
     println!("Token ID already set during canister initialization: {}", ledger_id);
 
-    // User calls get_deposit_subaccount
+    // Step 1: User calls get_deposit_subaccount
```

Comment:
Also, add another test to verify, the deposit function will fail, if user didn't make any transfer.